client/my-sites/themes/theme-options.js

```diff
@@ -113,7 +120,10 @@ const tryandcustomize = {
 			// is less readily available since it needs to be fetched using the `QuerySitePlans` component.
 			( isJetpackSite( state, siteId ) && ! isPremiumThemeAvailable( state, themeId, siteId ) )
 		) ||
-		( isJetpackSite( state, siteId ) && ! isThemeAvailableOnJetpackSite( state, themeId, siteId ) )
+		( isJetpackSite( state, siteId ) && ! isThemeAvailableOnJetpackSite( state, themeId, siteId ) ) ||
+		( config.isEnabled( 'jetpack/pijp' ) &&
+			isJetpackSite( state, siteId ) &&
+			! hasFeature( state, siteId, FEATURE_UNLIMITED_PREMIUM_THEMES ) )
```

Comment:
I just had a look at the code above this change, and I'm afraid this needs some more tweaking (damn complex `hideForTheme` criteria! 😄). We have a check for `isPremium` and `isJetpackSite` [further up](https://github.com/Automattic/wp-calypso/pull/13621/files#diff-b4173dc1d07244fce42a671d04c5c1caR139) that already uses `isPremiumThemeAvailable`, which [in turn checks](https://github.com/Automattic/wp-calypso/blob/9d783e9fe23cef87841d1aac1433c20d92dd964c/client/state/themes/selectors.js#L606) `hasFeature( ... )` for us.

So I think we can drop this change, and add a `config.isEnabled( 'jetpack/pijp' )` criterion to R139, i.e.

```js
( isJetpackSite( state, siteId ) && ! ( config.isEnabled( 'jetpack/pijp' ) && isPremiumThemeAvailable( state, themeId, siteId ) ) )
```

(I hope I got the parents right 😄 )